libs/design/src/atoms/loading-icon/loading-icon.component.ts

```diff
@@ -17,13 +17,14 @@ const _daffLoadingIconBase = daffColorMixin(DaffLoadingIconBase, 'primary')
   changeDetection: ChangeDetectionStrategy.OnPush,
   host: {
     '[style.max-width]': 'diameter + "px"',
```

Comment:
Can this be also done with `Hostbinding`?

Reply:
done